### PR TITLE
BF: Redundant assignment

### PR DESCRIPTION
**Pull request by boazmohar**
Another static code analysis PR.

### CODE REVIEW

File: psychopy/data/staircase.py [hunk 1]
```diff
@@ -858,9 +858,6 @@ def __init__(self,
             maxVal=maxVal, name=name, autoLog=autoLog)
 
         self.stopInterval = stopInterval
-
-        startVal = startVal
-        startValSd = startValSd
```
Comment (hoechenberger) [inline psychopy/data/staircase.py]: These lines should _not_ be removed; instead, the values should be assigned to the attributes `self.startVal` and `self.startValSd`, as it probably was meant to be all along.

We should also add a unit test for this.

Comment (boazmohar) [reply psychopy/data/staircase.py]: I have corrected that and added two assert statements to TestQuestHandler, is that ok?

**Comment by boazmohar** [comment]
Weird, that passes on my [Travis](https://travis-ci.org/boazmohar/psychopy/builds/317387838)

**Comment by hoechenberger** [comment]
@boazmohar Thanks for addressing to points I raised! I currently don't have time to check why the Travis tests are failing, though… Looks like a problem related to visual testing, not to any of the changes you've made.

**Comment by peircej** [comment]
FYI, I think either solution with the `startVal` and `startValSD` would have been fine - the value is actually used any further once it's passed to the underlying Quest object but I guess it's nice to keep it there anyway

**Pull request merged**